spikeinterface/comparison/hybridspikesrecording.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Union
+import numpy as np
+from spikeinterface.core import BaseSorting, NumpySorting, WaveformExtractor
+from spikeinterface.comparison import AddTemplatesRecording
+
+
+class HybridSpikesRecording(AddTemplatesRecording):
+
+	def __init__(self, wvf_extractor: WaveformExtractor, max_n_spikes: int = 1000,
+				 max_frac_spikes: float = 0.05, refractory_period: float = 1.5) -> None:
+		target_recording = wvf_extractor.recording
+		target_sorting = wvf_extractor.sorting
+		templates = wvf_extractor.get_all_templates()
+		injected_sorting = self._generate_injected_sorting(target_sorting, recording.get_num_frames(),
+						   max_n_spikes, max_frac_spikes, refractory_period)
+
+		AddTemplatesRecording.__init__(target_sorting, templates, wvf_extractor.nbefore, target_recording=target_recording)
+
+
+	# TODO: May be a better way of creating this?		
+	def _generate_injected_sorting(sorting: BaseSorting, t_max: int, max_n_spikes: int,
```

Comment:
I would make a clear function for this outside this class.
For instance in core/testing_tools.py have already some sorting generator (very very simple) this could be enhance and reanmed.

Reply:
@samuelgarcia I changed `core.generate_sorting` a bit, but I don't know how to implement the check if already existing spikes in a way that's elegant...
Any idea?